src/sentry/static/sentry/app/views/events/utils/eventsRequest.tsx

```diff
@@ -327,11 +338,10 @@ class EventsRequest extends React.PureComponent<EventsRequestProps, EventsReques
   /**
    * Transforms query response into timeseries data to be used in a chart
    */
-  transformTimeseriesData(data: EventsStatsData): Series[] {
-    const seriesName = this.props.currentSeriesName ?? 'Current';
+  transformTimeseriesData(data: EventsStatsData, seriesName?: string): Series[] {
     return [
       {
-        seriesName,
+        seriesName: seriesName || 'Current',
```

Comment:
Why drop null coalescing operator?

Reply:
I wanted to squash away 0 and '' as bad values too.